templates/slim/document.html.slim

```diff
@@ -10,12 +10,12 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
     meta content="yes" name="apple-mobile-web-app-capable"
     meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"
     meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui" name="viewport"
-    link href="#{revealjsdir}/css/reveal.min.css" rel="stylesheet"
+    link href="#{revealjsdir}/css/reveal.css" rel="stylesheet"
     / Default theme required even when using custom theme
     - if attr? :revealjs_customtheme
       link rel='stylesheet' href=(attr :revealjs_customtheme) id='theme'
     - else
-      link rel='stylesheet' href='#{revealjsdir}/css/theme/default.css' id='theme'
+      link rel='stylesheet' href='#{revealjsdir}/css/theme/black.css' id='theme'
```

Comment:
Black is the new default, it's true :wink: 
But serious, the default theme isn't there anymore in 3.0 (https://github.com/hakimel/reveal.js/tree/master/css/theme), black is now the default